addons/survey/models/survey_question.py

```diff
@@ -730,12 +759,13 @@ def _compute_display_name(self):
                 answer.display_name = answer.value
                 continue
             title = answer.question_id.title or _("[Question Title]")
-            n_extra_characters = len(title) + len(answer.value) + 3 - self.MAX_ANSWER_NAME_LENGTH  # 3 for `" : "`
+            answer_txt = answer.value or answer.value_image_filename or ''
```

Comment:
Can't we use `get_answer_label` here to avoid potentially multiple labels as `"What is the answer? : "`?
(This is used for the `triggering_answer_ids` field on the survey question form view - https://github.com/odoo/odoo/pull/146624/files#diff-561b4432aec8ae5a6ada609e7e6ffb2d5cdbfa8746700ebb871b87a7172849ccL225).

Reply:
What bothers me is the  `or ''`. In which cases isn't there any value then? Actually, seeing that there is a constraint, there shouldn't be such a case?

And it does seem like a good idea to have filenames in the "offline" results (the aggregate report). To check with François?